snippets/react-snippets.code-snippets

```diff
@@ -12,6 +12,15 @@
         ],
         "description": "Adds aria attributes for a dialog. A dialog is a window separate from the rest of the webpage like a modal."
     },
+    "SearchAria":{
+        "prefix": [
+            "SearchAria",
+        ],
+        "body": [
+            "role='search'",
```

Comment:
any chance we can also add role='searchbox' in this pr!

Reply:
sure